src/components/ControlPane.tsx

```diff
@@ -35,6 +41,16 @@ export class ControlPane extends React.Component<IControlPaneProps, IControlPane
   }
 
   private renderListPane(): React.ReactNode {
-    return this.state.isExpanded ? <div className="list-pane" /> : null;
+    return this.state.isExpanded 
+      ? <ListPane albums={this.props.albums} /> : null;
+  }
+
+  private handleKeyPress(event: KeyboardEvent): void {
+    if (this.state.isExpanded) {
```

Comment:
reduce branching: `if (!this.state.isExpanded && event.charCode >= 33 && event.charCode <= 126)`

Reply:
I don't want too many expressions in a single if statement, and this seems like a logical grouping.